### ISSUE TITLE
Disabling unused interfaces killing VPN

### ISSUE THREAD

**Issue by petiepooo** [Security-Onion-Solutions/securityonion#2970]
https://github.com/Security-Onion-Solutions/securityonion/blob/aa15f3ca4a00acd7f6d0af21c3627520f608f8d2/setup/so-functions#L58 still disables VPN interfaces during install.  Issue #1139 only fixed the startup issue, and IMHO is still doing it poorly.

I don't think white-listing NICs by name or prefix is the solution; I think you should not be disabling NICs at all.  NIC names are a matter of convention, but could be aliased to whatever a person desires.  I could just as well alias my VPN interfaces to vpn0, or naming them after the location to which they connect.  Because you're whitelisting based on name prefix, they would still fall through this filter and get disabled.

Furthermore, if a user has already setup a static IP as recommended for the admin interface, then I think you should assume that they will have already configured the remaining NICs, save for the monitoring interfaces, as they want them.

Fundamentally, why should unused NICs be disabled by the install script at all?  What is the problem with leaving unused NICs as is?  I think this is a solution to a non-existing problem that is itself more problem than what it was supposed to fix.

By all means, disable IPv6, for now, but recognize that's going to break things for some people too as IPv6 becomes more common.  But I recommend you rip out the whole section where you disable NICs that you believe to be unused simply because they're not a monitoring or admin interface.

**Comment by TOoSmOotH** [comment]
Disabling nics was due to users having issues with NICs that were online with dhcp addresses. This would cause install issues. We have since changed the logic on how to detect this and it might be safe now to not disable NICs. We will look into it in 2.3.40. In 2.3.30, the VPN config is tied to the defaults for wireguard (wgX) and openvpn (tunX) for now. This dependency should go away once we are not disabling unused NICs.
